### ISSUE TITLE
mk48io.com

### ISSUE THREAD

**Issue by finnbear** [SoftbearStudios/mk48#8]
Domain has been registered, must be configured.

**Comment by thatretrodev** [comment]
I find calling a game `[Game Name].io` and then using the domain `[Game Name]io.com` to be quite lazy. At the very most, you should have a domain called `[Game Name]io.com`, but make it redirect to `[Game Name].io`

**Comment by finnbear** [comment]
> I find calling a game `[Game Name].io` and then using the domain `[Game Name]io.com` to be quite lazy. At the very most, you should have a domain called `[Game Name]io.com`, but make it redirect to `[Game Name].io`

The main and only reason to have a .com domain is if organizations block .io domains by default, but not .com. There is no plan to switch domains.
